packages/calypso-build/webpack/minify.js

```diff
@@ -5,6 +5,7 @@ const TerserPlugin = require( 'terser-webpack-plugin' );
 const browserslist = require( 'browserslist' );
 const babelPlugins = require( '@babel/compat-data/plugins' );
 const semver = require( 'semver' );
+const CssMinimizerPlugin = require( 'css-minimizer-webpack-plugin' );
```

Comment:
This seems like a change that deserves a mention in the CHANGELOG of `calypso-build`, and maybe a major release. There are already some unreleased breaking changes made, I guess it would be a good time to release them.

The same applies to `webpack-rtl-plugin`.

Reply:
Done.